### PR TITLE
[zh] Fix some 404 links

### PR DESCRIPTION
**Pull request by ydFu**
1. Update out-of-sync content in English
2. Fix 404 links.
3. Fix fail render.
image

As follows:
```
1. content/zh-cn/docs/concepts/security/pod-security-policy.md
2. content/zh-cn/docs/reference/glossary/host-aliases.md
3. content/zh-cn/docs/reference/glossary/kubelet.md
```

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

**Review by howieyuen — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/concepts/security/pod-security-policy.md [hunk 1]
```diff
@@ -620,9 +622,9 @@ kubectl-user get pod pause -o yaml | grep kubernetes.io/psp
 ```
 
 <!--
-The output is similar to this:
+The output is similar to this
 -->
-输出类似于：
+输出类似于
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/pod-security-policy.md:627]:
```suggestion
输出类似于：
```
The original colon `：` is better. It is not necessary to follow the en here in my opinion.

File: content/zh-cn/docs/reference/glossary/kubelet.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Kubelet
 id: kubelet
 date: 2018-04-12
-full_link: /zh-cn/docs/reference/generated/kubelet
+full_link: /docs/reference/generated/kubelet
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/glossary/kubelet.md:5]: The en link is redirected to `/docs/reference/command-line-tools-reference/kubelet/`.

Whether the zh-cn link should follow? We may need to modify this file:
https://github.com/kubernetes/website/blob/main/static/_redirects

File: content/zh-cn/docs/concepts/security/pod-security-policy.md (hunk 1 above)
Comment (ydFu) [reply content/zh-cn/docs/concepts/security/pod-security-policy.md:627]: I think if the upstream is not good and cannot express the correct meaning, it should be changed upstream.

If each downstream is translated with a special case, the contributors and reviewers will have no basis to become their own creations, which will make it difficult for subsequent people to maintain.

Comment (tengqm) [reply content/zh-cn/docs/concepts/security/pod-security-policy.md:627]: Adding '：' is a good suggestion, I think.

**Comment by taniaduggal** [comment]
LGTM

**Pull request merged**